src/app/homepage/homepage.component.ts

```diff
@@ -105,6 +106,10 @@ export class HomepageComponent implements OnInit, OnDestroy, AfterViewInit {
       return;
     }
 
+    if (carbonRef) {
+      sponsorsWrapper.classList.add('sponsors-carbon');
```

Comment:
When is this class removed (upon what conditions)? Like, what if you resize a window?

Reply:
If there is a banner then it is there and it appears, if not then no

Reply:
![1](https://user-images.githubusercontent.com/26163841/108683384-3680f880-74fa-11eb-909f-6afdf30c3c27.jpg)
after fix, before in issue image https://github.com/nestjs/docs.nestjs.com/issues/1790